runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -244,7 +252,8 @@
                 try {
                     router.addClient(new LayoutClient())
                             .addClient(new SequencerClient())
-                            .addClient(new LogUnitClient())
+                            .addClient(new LogUnitClient().setMetricRegistry(metrics != null
+                                            ? metrics : CorfuRuntime.getDefaultMetrics()))
```

Comment:
Yes long term plan is to introduce annotations like the ones used in MP. @Timed, @Metered, etc.